crates/matrix-sdk-crypto/src/session_manager/group_sessions/share_strategy.rs

```diff
@@ -746,7 +746,12 @@ mod tests {
     fn test_serialize_device_based_strategy() {
         let encryption_settings = all_devices_strategy_settings();
         let serialized = serde_json::to_string(&encryption_settings).unwrap();
-        assert_snapshot!(serialized);
+        with_settings!({ sort_maps => true, prepend_module_to_snapshot => false }, {
+            assert_snapshot!(
+                "serialize_device_based_strategy",
```

Comment:
there is no need to specify an explicit snapshot name here.

```suggestion
```

Likewise below.

Reply:
But original filename `matrix_sdk_crypto__session_manager__group_sessions__share_strategy__tests__serialize_device_based_strategy.snap` was too long and I just wanted to reduce it for windows env.

Reply:
yes. You did that with `prepend_module_to_snapshot => false`.

Reply:
Ah, understood.
Fixed by https://github.com/matrix-org/matrix-rust-sdk/pull/4625/commits/bce99c272a2d3d620f813e6ab66b439d296b9645